#!/home/venice/Documents/irri-galaxy-dev/database/dependencies/_conda/bin/python
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
